avae/train.py

```diff
@@ -191,9 +246,6 @@ def train(
         recon_fn=recon_fn,
     )
 
-    t_history = []
-    v_history = []
-
     print(
         "Epoch: [0/%d] | Batch: [0/%d] | Loss: -- | Recon: -- | "
         "KLdiv: -- | Affin: -- | Beta: --" % (epochs, len(trains)),
```

Comment:
Might be a good idea to also add cyclical gamma to the feedback printout since beta is already here

Reply:
Ok I've now found a separate printout related to this in the #140 PR (line 440). That's fine we can have a separate one, but in that case, remove beta from here. No need to print it twice.